pkg/metrics/init.go

```diff
@@ -54,6 +56,7 @@ type CoreSchedulerMetrics interface {
 	// Metrics Ops related to ScheduledAllocationSuccesses
 	IncAllocatedContainer()
 	AddAllocatedContainers(value int)
+	getAllocatedContainers() *prometheus.Counter
```

Comment:
can we not simply return the primitive we need?
Hide all the prometheus logic inside one file, see other review comments on how.

Reply:
Makes sense, fixed